src/main/java/org/sonar/plugins/stash/issue/collector/StashCollector.java

```diff
@@ -153,6 +193,37 @@ public static StashDiffReport extractDiffs(String jsonBody) throws StashReportEx
                   }
                 }
               }
+              
+              // Extract File Comments: this kind of comment will be attached to line 0
+              JSONArray jsonLineComments = (JSONArray) jsonDiff.get("fileComments");
+              if (jsonLineComments != null) {
+                
+                StashDiff initialDiff = new StashDiff(StashPlugin.CONTEXT_ISSUE_TYPE, path, 0, 0);
+                          
+                for (Object objLineComment : jsonLineComments.toArray()) {
+                  JSONObject jsonLineComment = (JSONObject) objLineComment;
+                              
+                  long lineCommentId = (long) jsonLineComment.get("id");
+                  String lineCommentMessage = (String) jsonLineComment.get("text");
+                  long lineCommentVersion = (long) jsonLineComment.get("version");
+                              
+                  JSONObject objAuthor = (JSONObject) jsonLineComment.get("author");
+                  if (objAuthor != null) {
+                                  
+                    long authorId = (long) objAuthor.get("id");

```

Comment:
doesn't it make sense to use extractUser()?
https://developer.atlassian.com/static/rest/stash/3.11.3/stash-rest.html#idp1467264
emailAddress insteadof email, no?